ros/api/v1/hosts.py

```diff
@@ -466,6 +500,29 @@ def get(self):
         io_states_dict['undersized'] = -1
         total_conditions = totals['cpu'] + totals['memory'] + totals['io']
 
+        current_utc_datetime = datetime.now(timezone.utc)
+        upload_date = (current_utc_datetime - timedelta(days=1)).date()
+
+        current_performance_profiles = [
+            record
+            for record in systems_with_performance_record_queryset
+            if record.report_date.date() == upload_date
+        ]
+
+        historical_performance_profiles = db.session.query(
+            PerformanceProfile).select_entity_from(
```

Comment:
Instead of `select_entity_from()` can we use `aliased()`.  `select_entity_from()` is deprecated and will be removed from SQLAlchemy 2.0. 
Check the deprecation note - https://docs.sqlalchemy.org/en/14/orm/query.html#sqlalchemy.orm.Query.select_entity_from

Reply:
@patilsuraj767 Sure I will check, btw is there a way we can check these warnings?
Did this one just show up on the terminal?

Reply:
@patilsuraj767 Change pushed. I would like squash the commits before rebase and merge.